guides/tutorials/codelabs/oracle-kubernetes-source-to-prod/index.md

```diff
@@ -29,24 +29,24 @@ SSH onto the compute instance you previously created.  `IP` is its public ip add
 ssh ubuntu@IP
 ```
 
-Install Halyard.
+#### Install Halyard
```

Comment:
Only the curl command is for Installing Halyard.  The rest of the content are about configuring Halyard.

Reply:
Actually the rest is about configuring Spinnaker.